export.py

```diff
@@ -721,7 +721,9 @@ def run(
     if half:
         assert device.type != 'cpu' or coreml, '--half only compatible with GPU export, i.e. use --device 0'
         assert not dynamic, '--half not compatible with --dynamic, i.e. use either --half or --dynamic but not both'
-    model = attempt_load(weights, device=device, inplace=True, fuse=True)  # load FP32 model
+    
+    exp_yolo_fastest = True
+    model = attempt_load(weights, device=device, inplace=True, fuse=True, exp_yolo_fastest=exp_yolo_fastest)  # load FP32 model
```

Comment:
지금 repo는 yolo fastest만을 위한 repo 이므로 726 line에 바로 `exp_yolo_fastest=True`로 넣던지 `attempt_load` 내부에서 `exp_yolo_fastest=True`로 넣어도 되지 않을까요?

Reply:
좋은 질문입니다. 같은 고민을 해봤던 부분입니다. 처음에는 신국님 말씀하신 것처럼 수정했는데, 그런데 train.py 469줄 best.pt를 validation할 때 attemp_load함수를 호출합니다. 그래서 exp_yolo_fastest는 들어가야 될 것 같은데 arguments에 넣을 수도 있고 그냥 이렇게 작성한 것은 yolo-fastest만 export해서 default하게 만들었습니다. 혹시 어떻게 진행하면 좋을까요?

Reply:
그러면 attempt_load 내부는 건드리지 않아야겠네요. 아래처럼 export할때만 동작하도록 아래처럼 수정하면 어떨까요?
```suggestion
    model = attempt_load(weights, device=device, inplace=True, fuse=True, exp_yolo_fastest=True)  # load FP32 model
```

Reply:
네네 좋습니다. 그렇게 수정하도록 하겠습니다.

Reply:
말씀하신 대로 수정하였습니다.